dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxws.3.2/src/org/apache/cxf/jaxws/ServiceImpl.java

```diff
@@ -133,13 +132,25 @@ public ServiceImpl(Bus b, URL url, QName name, Class<?> cls, WebServiceFeature .
     }
 
     final void initialize(Bus b, URL url, WebServiceFeature ... f) {
+        
+        boolean isFinestEnabled = LOG.isLoggable(Level.FINEST);   // Liberty Change issue #26529
         if (b == null) {
             b = BusFactory.getThreadDefaultBus(true);
+            if (isFinestEnabled) {
+                LOG.finest("Bus is not provided as initialization parameter. It's obtained from default bus on the thread: " + b); // Liberty Change issue #26529
```

Comment:
We probably don't need trace at lines 140 and 144 as entry trace for initialize() method should log value of Bus b.
As a general comment, we should avoid logging duplicate information to reduce total amount of trace.